tests/test_gcloud.py

```diff
@@ -262,6 +263,65 @@ def test_from_settings_with_defaults():
 
         assert settings["storage.gcloud.bucket_name"] in bucket_options
 
+
+def test_from_settings_with_defaults_and_uniform_bucket_level_access_enabled():
+    from pyramid_storage import gcloud
+
+    settings = {
+        "storage.gcloud.bucket_name": "Attachments",
+        "storage.gcloud.uniform_bucket_level_access": True
+    }
+
+    inst = gcloud.GoogleCloudStorage.from_settings(settings, "storage.")
+
+    assert inst.bucket_name == settings["storage.gcloud.bucket_name"]
+    assert inst.uniform_bucket_level_access == settings["storage.gcloud.uniform_bucket_level_access"]
+    assert inst.acl == None
+    assert inst.auto_create_acl == None
+
+
+def test_from_settings_with_defaults_and_uniform_bucket_level_access_enabled_as_string():
+    from pyramid_storage import gcloud
+
+    settings = {
+        "storage.gcloud.bucket_name": "Attachments",
+        "storage.gcloud.uniform_bucket_level_access": "true"
+    }
+
+    inst = gcloud.GoogleCloudStorage.from_settings(settings, "storage.")
+
+    assert inst.bucket_name == settings["storage.gcloud.bucket_name"]
+    assert inst.uniform_bucket_level_access == asbool(settings["storage.gcloud.uniform_bucket_level_access"])
```

Comment:
Here you can assume `True` (without using `asbool()`)

so basically `assert inst.uniform_bucket_level_access`